### ISSUE TITLE
How do you do preprocessing?

### ISSUE THREAD

**Issue by zhangrong1722** [Nomikxyz/diabetic-retinopathy-screening#3]
Your work is great.I check your script retrain.py.And I am interested in data preprocessing because the size of images in the dataset is extremely big,most of which are larger than 1000*1000 actually.Could you tell me how you do image preprocessing?

Thanks.

**Comment by javathunderman** [comment]
Hi, there isn't much pre-processing to do. Inception (the software that does the image processing) is pretty flexible and I was able to use the files from the [Kaggle dataset](https://www.kaggle.com/c/diabetic-retinopathy-detection/data) without much hassle. You can use Nomikxyz/retinopathy-dataset to get a smaller, curated set of images, or you can use [this](https://gist.github.com/javathunderman/6a9f9c8a0ee29cb7a817777421685e49) script to move the files from the Kaggle dataset into neatly sorted directories by using the provided classifications (in the trainLabels.csv file)

Let me know if you need more help!

**Comment by zhangrong1722** [comment]
Hi,
[This](https://gist.github.com/javathunderman/6a9f9c8a0ee29cb7a817777421685e49) script you refer to splits the Kaggle dataset into two classes by the provided classifications.I focus on the dataset preprocessing because the images in the dataset are too larger,most of whose size are larger than 2000x2000 while recent neural networks'(e.g. vgg,resnet) input are very small(e.g. 224*224).Intuitional,I can resize original images to specified scalar.But I afraid this may results in low-resolution and information loss because of severe downsampling.Could you give me same suggestion?And how does it do about the fact that Inception network does the image preprocessing?

Thanks.

**Comment by javathunderman** [comment]
Sorry, I'm not sure how to do that. My understanding of Tensorflow and ML in general is pretty rudimentary and I did very minimal pre-processing on this project, so I'm not sure how to scale it down like that. You can try posting this on subreddits like [/r/machinelearning](http://reddit.com/r/machinelearning) or [/r/computervision](http://reddit.com/r/computervision). The people on those forums are more knowledgeable than I am on pre-processing. 

If you need further help with using the project, please let me know. I'll try to answer the best I can.

**Comment by zhangrong1722** [comment]
thanks.

**Comment by javathunderman** [comment]
@estelle1722 I'm closing the issue for now, unless you need more help.
